src/passport-saml/saml.ts

```diff
@@ -733,6 +733,7 @@ class SAML {
       getKeyInfo: key => "<X509Data></X509Data>",
       getKey: keyInfo => Buffer.from(this.certToPEM(cert)),
     };
+    signature = this.normalizeNewlines(signature.toString());
```

Comment:
You can verify this fix by removing this line and running `test-signatures.js`. The `CRLF line endings` and `CR line endings` tests will fail, while the `R1A - both signed => valid` will pass, even though all three tests use the same fixture. This normalization of newlines passes all three tests.